src/services/session.service.js

```diff
@@ -48,6 +49,12 @@ module.exports = (() => {
   }
 
   function authenticateSession(req, res, next) {
```

Comment:
authenticateSession also sets the current user in session which will be needed in some api calls.